iterators.tex

```diff
@@ -6349,8 +6356,8 @@
 \oldtxt{\tcode{begin} and \tcode{end} are required to be amortized constant time.}
 
 \pnum
-\newtxt{Given an object \tcode{t} of type \tcode{T}, \oldtxt{the type \tcode{T} models}
-\tcode{Range\newtxt{<T>()}} \newtxt{is satisfied} if and only if}
+\newtxt{Given a value \tcode{t} of type \tcode{T}, \tcode{Range<T>()} is satisfied
+if and only if}
```

Comment:
> What's the rationale for changing object to value? Can you have a "value" of a reference type?

You certainly can't have an object of a reference type. You're right, "value" isn't much better.

> Also, you have mismanaged the markup here.

I removed the nested markup: the whole sentence is `\newtxt`, I didn't think it necessary to have nested tags. The entire sentence is new since N4382, and to my knowledge hasn't received any review.